tests/test_spidermiddleware.py

```diff
@@ -124,9 +124,7 @@ def _construct_mw_setting(*mw_classes, start_index: Optional[int] = None):
         return {i: c for c, i in enumerate(mw_classes, start=start_index)}
 
     def _scrape_func(self, *args, **kwargs):
-        yield {'foo': 1}
-        yield {'foo': 2}
-        yield {'foo': 3}
+        yield from ({'foo': 1}, {'foo': 2}, {'foo': 3})
```

Comment:
I am not sure this is better than before.

Reply:
I would say it is "better".  In the sense that moving these items from multi-line to one line does not make it any more difficult to understand while performing the same task using a new language feature.